src/backend/core/models.py

```diff
@@ -751,6 +750,18 @@ class Label(BaseModel):
         help_text=_("Threads that have this label"),
         blank=True,
     )
+    description = models.CharField(
+        _("description"),
+        max_length=255,
+        blank=True,
+        default="",
+        help_text=_("Description of the label, used by AI to understand its purpose"),
+    )
+    is_auto = models.BooleanField(
+        _("auto labeling"),
+        default=True,
```

Comment:
IMO this should be disabled by default
```suggestion
        default=False,
```